docs/HyperIndex/migrate-with-ai.md

```
@@ -0,0 +1,81 @@
+```
+
+Follow the CLI prompts to set up the boilerplate indexer with the same contracts and events as your existing subgraph.
+
+:::caution
+The Claude migration skills are only available in HyperIndex v3. If the latest stable version is not v3, you need to specify a v3 version explicitly:
+
+```bash
+pnpx envio@3.0.0-alpha.21 init
+```
+
+Replace `3.0.0-alpha.21` with the latest available v3 version. You can find the latest releases [here](https://github.com/enviodev/hyperindex/releases).
```

markdownlint-cli2 (0.22.0)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)